src/runtime_src/core/pcie/driver/windows/include/XoclUser_INTF.h

```diff
@@ -185,6 +185,14 @@ typedef struct _XOCL_MAP_BAR_RESULT {
     ULONG       BarLength;     // OUT: Length of mapped buffer
 } XOCL_MAP_BAR_RESULT, *PXOCL_MAP_BAR_RESULT;
 
+typedef struct _XOCL_ROM_INFORMATION {
```

Comment:
Don't change driver headers here.   This change must be completely reverted.
I think the structure you define here is avaiable in a header under runtime_src/core/include.  Please check.

The XoclUser_INTF.h is not owned by the XRT source tree; it is copied from driver's github repo to here.  Please coordinate any changes to this file with Arpit and Igor.  No changes should be necessary and must definitely not percolate into XRT general code as is done in this PR.

Reply:
This header will be used in xocl user driver also as part of ROM IOCTL. So, added it here.

Reply:
Please check the corresponding device driver change at https://github.com/Xilinx/MS_XOCLUSER/blob/master/Src/Inc/XoclUser_INTF.h#L337